src/core/server/graph/resolvers/Comment.ts

```diff
@@ -64,6 +64,13 @@ export const Comment: GQLCommentTypeResolver<comment.Comment> = {
 
     return canModerate(ctx.user, { siteID: c.siteID });
   },
+  canReply: (c) => {
+    if (typeof c.rejectedAncestor === "undefined") {
+      return true;
+    }
+
+    return !c.rejectedAncestor;
```

Comment:
This is so clever, I didn't even know there was a `rejectedAncestor` computed prop on comments.